### ISSUE TITLE
Image reference for FaceMesh

### ISSUE THREAD

**Issue by alanvww** [ml5js/ml5-website-v02-docsify#178]
Similar to #176 , it will be great if we can include a ml5 version of [this diagram](https://github.com/tensorflow/tfjs-models/blob/master/face-landmarks-detection/mesh_map.jpg) in our docs website. @MOQN @QuinnHe @myrahsa

**Comment by shiffman** [comment]
Thanks for posting this @alanvww! I'll also note that when going over this model in class, I found that the index numbers are very hard to read. I realize it's probably quite a challenge to make a clear design for this diagram given the number of points in the mesh, but I think it could be very valuable!

**Comment by MOQN** [comment]
Thank you @alanvww and @shiffman! Let me work on it this week (or weekend)!

**Comment by leey611** [comment]
texture map examples like [this](https://editor.p5js.org/codingtrain/sketches/zUKp9n4MW) is a helpful demonstration for implementing a custom face mask. @alanvww and i would like to add [the diagram](https://github.com/tensorflow/tfjs-models/blob/master/face-landmarks-detection/mesh_map.jpg) under `faceMesh.getUVCoords()` as a reference for creating custom masks.

**Comment by alanvww** [comment]
> texture map examples like [this](https://editor.p5js.org/codingtrain/sketches/zUKp9n4MW) is a helpful demonstration for implementing a custom face mask. @alanvww and i would like to add [the diagram](https://github.com/tensorflow/tfjs-models/blob/master/face-landmarks-detection/mesh_map.jpg) under [`faceMesh.getUVCoords()`](https://docs.ml5js.org/#/reference/facemesh?id=facemeshgetuvcoords) as a reference for creating custom masks.

@shiffman @MOQN let us know your thoughts! I’m also working on a sketch to show the mesh map in a more interactive/detailed way. I’ll post update here soon.

**Comment by alanvww** [comment]
> > texture map examples like [this](https://editor.p5js.org/codingtrain/sketches/zUKp9n4MW) is a helpful demonstration for implementing a custom face mask. [@alanvww](https://github.com/alanvww) and i would like to add [the diagram](https://github.com/tensorflow/tfjs-models/blob/master/face-landmarks-detection/mesh_map.jpg) under [`faceMesh.getUVCoords()`](https://docs.ml5js.org/#/reference/facemesh?id=facemeshgetuvcoords) as a reference for creating custom masks.
> 
> [@shiffman](https://github.com/shiffman) [@MOQN](https://github.com/MOQN) let us know your thoughts! I’m also working on a sketch to show the mesh map in a more interactive/detailed way. I’ll post update here soon.

Sorry for the delay! This is the sketch I mentioned: https://editor.p5js.org/alanvww/sketches/AlRRdSTW9 Feel free to give me any feedback :D

**Comment by shiffman** [comment]
I agree we should include the fesh mesh diagram into the documentation! I also really like the sketch highlighting the parts! Fantastic! 

Is the sketch meant to be an example or more of a "demo"? There are aspects of the way the code is written (`forEach` etc)  that may be unfamiliar to p5.js + ml5.js users. Also, should these custom lists of points be included as part of what we return to the model `results`?

**Comment by MOQN** [comment]
> The custom lists of keypoints were actually created by me for my ongoing project.

I am curious! Haha.

**Comment by jackbdu** [comment]
@alanvww I had no idea you already created a sketch for this. I just made my version of an [interactive mesh map for FaceMesh](https://editor.p5js.org/jackbdu/sketches/HGCFwtbJZ). Feel free to share any feedback or customize however you'd like.

**Comment by alanvww** [comment]
> [@alanvww](https://github.com/alanvww) I had no idea you already created a sketch for this. I just made my version of an [interactive mesh map for FaceMesh](https://editor.p5js.org/jackbdu/sketches/HGCFwtbJZ). Feel free to share any feedback or customize however you'd like.

Heyyyy @jackbdu, I think your sketch makes perfect sense! Would you like to create a PR to add it to the docs and potentially replace the diagram 👀?
